doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -497,6 +497,16 @@ Modem libraries
       * Removed dependency on the :ref:`lib_modem_jwt` library.
         The :ref:`lib_location` library now selects :kconfig:option:`CONFIG_NRF_CLOUD_REST_AUTOGEN_JWT` when using :kconfig:option:`CONFIG_NRF_CLOUD_REST`.
 
+    * Removed:
+
+       * Skyhook support removed.
```

Comment:
```suggestion
       * Support for Skyhook.
```
The bullet above already suggests that something is removed.

Reply:
True. Changed